configure.ac

```diff
@@ -1954,6 +1967,11 @@ fi
 case $add_players in
   *omxplayer*)
       XB_ADD_PLAYER([OMXPLAYER], [omxplayer])
+      AC_DEFINE([TARGET_RASPBERRY_PI],[1],["Define to 1 for RASPBERRY_PI"])
```

Comment:
I see. Although I don't particularly like it, other distros might have similar issues.
@popcornmix your decision
In the longer run, it might be sensible to rename TARGET_RASPBERRY_PI to something more generic.